### PR TITLE
Update Parallel.ForEachAsync.cs

### PR DESCRIPTION
**Pull request by theodorzoulias**
Correction of a few errors in the xml comments of the Parallel.ForEachAsync method.

### CODE REVIEW

**Comment by dnfadmin** [comment]
[CLA assistant check](https://cla.dotnetfoundation.org/dotnet/runtime?pullRequest=64973) <br/>Thank you for your submission, we really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla.dotnetfoundation.org/dotnet/runtime?pullRequest=64973) before we can accept your contribution.<br/><br/>:x: theodorzoulias [sign now](https://cla.dotnetfoundation.org/dotnet/runtime?pullRequest=64973)<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla.dotnetfoundation.org/check/dotnet/runtime?pullRequest=64973) it.</sub>

File: src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/Parallel.ForEachAsync.cs [hunk 1]
```diff
@@ -251,9 +251,9 @@ public static Task ForEachAsync<TSource>(IAsyncEnumerable<TSource> source, Paral
             return ForEachAsync(source, parallelOptions.EffectiveMaxConcurrencyLevel, parallelOptions.EffectiveTaskScheduler, parallelOptions.CancellationToken, body);
         }
 
-        /// <summary>Executes a for each operation on an <see cref="System.Collections.Generic.IEnumerable{TSource}"/> in which iterations may run in parallel.</summary>
+        /// <summary>Executes a for each operation on an <see cref="System.Collections.Generic.IAsyncEnumerable{TSource}"/> in which iterations may run in parallel.</summary>
         /// <typeparam name="TSource">The type of the data in the source.</typeparam>
-        /// <param name="source">An enumerable data source.</param>
+        /// <param name="source">An async-enumerable data source.</param>
```
Comment (stephentoub) [inline src/libraries/System.Threading.Tasks.Parallel/src/System/Threading/Tasks/Parallel.ForEachAsync.cs:256]:
```suggestion
        /// <param name="source">An asynchronous enumerable data source.</param>
```

**Review by stephentoub — APPROVED** [review]
Thanks.

**Comment by theodorzoulias** [comment]
I closed this pull request. I assumed that I should close it since it was approved. Sorry if this is a problem, I am not very familiar with the GitHub procedures.

**Comment by stephentoub** [comment]
> I closed this pull request. I assumed that I should close it since it was approved. Sorry if this is a problem, I am not very familiar with the GitHub procedures.

Approval just means the changes look acceptable.  By closing it, we can't merge the changes.

**Comment by theodorzoulias** [comment]
@stephentoub oops! I messed up, sorry. I just created a [new pull request](https://github.com/dotnet/runtime/pull/65009), because I had also deleted my local copy of the runtime repository.

**Pull request closed without merging**